.github/workflows/_reusable.rust.yml

```diff
@@ -37,6 +37,33 @@ jobs:
     container:
       image: xaynetci/yellow:v9
     timeout-minutes: 20
+    env:
+      XAYN_INGESTION__NET__BIND_TO: 127.0.1.1:3030
+      INGESTION_URI: http://127.0.1.1:3030
+      XAYN_PERSONALIZATION__NET__BIND_TO: 127.0.1.1:3031
+      PERSONALIZATION_URI: http://127.0.1.1:3031
+    services:
```

Comment:
❓ if i understand the github docs correctly, then those services are always setup, even though we are not using them currently, which would make the ci unnecessarily slower. are the integration tests supposed to be run always with the ci in the future? then we of course need to have this. otherwise could we start the services also conditionally?

Reply:
AFIK there is no good way to start the services conditionally (nicely, there are hacky ways to run docker in docker, it's error prone and non trivial, while there are non-official actions to do so it would avoid it and if we do it it would still be limited to one step in which you both run docker and the integration tests).

What works is to have another job for integration tests, with the drawback of  now having to compile twice, through this might be anyway necessary.

I think we should have some discussion about when we run integration tests.